### PR TITLE
created schema and query file

### LINKED ISSUE #14: Create schema for database
### Task

- [x] Create schema based on ERD diagram 

### User Story 

As a developer, I want to upload data to a database based on a defined schema

### PR DESCRIPTION
**Pull request by A-Faris**
Added database folder with schema and query files
closes #14

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    operator_code CHAR(2) NOT NULL REFERENCES operator(operator_code),
+    disruption_info TEXT NOT NULL,
+    creation_time TIMESTAMP(0) NOT NULL,
+    incident_start TIMESTAMP(0) NOT NULL,
+    incident_end TIMESTAMP(0) NOT NULL,
+    is_planned BOOLEAN NOT NULL,
+    incident_summary TEXT NOT NULL,
+    incident_description TEXT NOT NULL,
+    incident_uri TEXT NOT NULL,
+    affected_routes TEXT NOT NULL
```
Comment (Zhi-704) [inline database/schema.sql]: How is disruption info different from the rest

File: database/schema.sql [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    booked_arrival TIMESTAMP(0) NOT NULL,
+    actual_arrival TIMESTAMP(0) NOT NULL,
+    booked_departure TIMESTAMP(0) NOT NULL,
+    actual_departure TIMESTAMP(0) NOT NULL,
+    operator_code CHAR(2) NOT NULL,
+    station_crs CHAR(3) NOT NULL REFERENCES station(station_crs)
+);
+
+CREATE TABLE cancellation(
+    cancellation_id BIGINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
```
Comment (Zhi-704) [inline database/schema.sql]: Would there really be enough incidents to warrant bigint?

File: database/schema.sql (hunk 1 above)
Comment (A-Faris) [reply database/schema.sql]: Removed it

File: database/schema.sql (hunk 2 above)
Comment (A-Faris) [reply database/schema.sql]: fixed, made it smallint

File: database/README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+- `station`: Stores information about railway stations.
+- `users`: Stores user information for the Railway Tracker application.
+- `cancel_code`: Stores codes and descriptions for cancellation reasons.
+- `incident`: Stores information about incidents and disruptions on the railway network.
+- `station_performance_archive`: Stores historical performance data for stations, including average delays and cancellation counts.
+- `subscription`: Stores user subscriptions to receive updates from specific operators.
+- `waypoint`: Stores information about scheduled and actual arrival and departure times for trains at stations.
+- `cancellation`: Stores information about cancelled train services, including the cancellation reason and associated waypoint.
+- `affected_operator`: Stores information about operators affected by a particular incident.
+
```
Comment (FarihaChoudhury) [inline database/README.md:59]: Good explanation

**Review by FarihaChoudhury — APPROVED** [review]
Good work making a schema

**Pull request merged**